layouts/_default/list.html

```diff
@@ -9,7 +9,7 @@
         {{ $paginator := .Paginate (where .Data.Pages "Section" "!=" "") }}
         <div>{{ range $paginator.Pages }}
           <div class="listItem">
-            <a class="itemImage project" href="{{ .RelPermalink }}" style="background-image: url(/articles/{{ .Param "slug"}}-sq.jpg);"></a>
+            <a class="itemImage project" href="{{ .RelPermalink }}" style="background-image: url({{ .RelPermalink }}{{ .Param "slug"}}-sq.jpg);"></a>
```

Comment:
Currently this uses the .RelPermalink and then the slug which is an exact match to the `*-sq.jpg` images. Would it be work trying to do something like with the banner on this? cc @apreshill

Reply:
A lot of this changed in the theme, here is the new default list layout:
https://github.com/batpigandme/tidyverse.org/blob/e8b0ba4c4b6fc4f1dd7665b11089813f0f8767d3/themes/hugo-tourmaline/layouts/_default/list.html#L14

And the current single layout:
https://github.com/batpigandme/tidyverse.org/blob/e8b0ba4c4b6fc4f1dd7665b11089813f0f8767d3/themes/hugo-tourmaline/layouts/_default/single.html#L4

So, instead of hard-coding the file paths I used the Hugo [page resources](https://gohugo.io/content-management/page-resources/) and matched based on regex (easily changed) of any image resource with the string `sq` in the title (for the list page, the thumbnail) or the string `wd` in the title (for the single page, the banner).

Reply:
Amazing!